src/app/components/projects/pages/details/details.component.spec.ts

```diff
@@ -43,9 +45,16 @@ describe("ProjectDetailsComponent", () => {
     imports: [SharedModule, MockBawApiModule],
     declarations: [mock.map, mock.card],
     component: DetailsComponent,
+    providers: [PageTitleStrategy],
   });
   const emptyResponse = [[]];
 
+  assertPageInfo(DetailsComponent, "test name", {
+    project: {
+      model: new Site(generateSite({ name: "test name" }))
```

Comment:
this is a project details page? Why are we generating a site model?

Reply:
This is a good catch. Surprisingly, this test didn't fail because of it.

I'll probably have to add some form of type checking to the assertion helper method to ensure this gets picked up in future tests